src/Compilers/CSharp/Portable/Emitter/EditAndContinue/CSharpSymbolMatcher.cs

```diff
@@ -736,6 +787,63 @@ private bool ArePointerTypesEqual(PointerTypeSymbol type, PointerTypeSymbol othe
                 return AreTypesEqual(type.PointedAtType, other.PointedAtType);
             }
 
+            private bool AreFunctionPointerTypesEqual(FunctionPointerTypeSymbol type, FunctionPointerTypeSymbol other)
+            {
+                var sig = type.Signature;
+                var otherSig = other.Signature;
+
+                if (!isEqual(sig.ReturnTypeWithAnnotations, otherSig.ReturnTypeWithAnnotations,
+                             sig.RefKind, otherSig.RefKind,
+                             sig.RefCustomModifiers, otherSig.RefCustomModifiers,
+                             allowOut: false))
+                {
+                    return false;
+                }
+
+                if (sig.ParameterCount != otherSig.ParameterCount)
```

Comment:
Consider replacing `if` and `for` loop with `if (sig.Parameters.SequenceEqual(otherSig.Parameters, (x, y) => isEqual(...)) { ... }`.